src/current/v22.2/backup-validation.md

```diff
@@ -84,7 +84,7 @@ SHOW TABLES FROM movr;
 (6 rows)
 ~~~
 
-You'll find the tables in place with no user table data.
+You'll find the tables in place with no rows.
```

Comment:
```suggestion
```
Maybe replace line 69 with: "Verify that the table schemas have been restored, but that the tables contain no rows: